autode/neb/original.py

```diff
@@ -361,59 +399,39 @@ class NEB:
 
     def __init__(
         self,
-        initial_species: Optional["autode.Species"] = None,
```

Comment:
I am not quite sure the new interface `from_end_points()` is better to someone not familiar with python or a non-programmer. Did you change the interface to make it consistent with `from_*` methods?

Reply:
> Did you change the interface to make it consistent with from_* methods?

Exactly that. You're totally right that it's less intuitive for those not used to named constructors. I'll add a deprecation exception with a pointer to the new interface 👍🏼